### ISSUE TITLE
Feature Request: Auto Upload delete pictures after a amount of time

### ISSUE THREAD

**Issue by nevyen** [nextcloud/ios#1677]
It would be a nice feature if the app would delete automatic uploaded pictures after a amount of time.

### Reasoning or why should it be changed/implemented?
Example for this use case: 
1. on vacation I take pictures and in the evening I load them up into the NC.
2. I get home after around 3 weeks and like to share them with some friends

For now I need to look in the paperbin for the uploaded pictures, If I like to share them I need to recreate the picture. Share it and delete it again. 

At worst case the picture is uploaded again.

### Environment data

**iOS version:** e.g. iOS 14.7.1

**Nextcloud iOS app version:** see More > Settings
4.0.6.0
**Server operating system:**
Ubuntu
**Web server:** Apache, nginx
Apache
**Database:**
MYSQL
**PHP version:**

**Nextcloud version:** see Nextcloud admin page
20.0.8

**Comment by marinofaggiana** [comment]
Hi, simple, when you go in vacation disable the delete auto upload and remove the photo manually

**Comment by eharris** [comment]
I disagree, I don't want the chance of my photos being deleted automatically ever.  I use the auto-upload as a fail-safe to make sure I have a copy of my files up on the server (in case my phone is lost or broken) even if I have forgotten to upload them somewhere else (like to an shared folder for a particular event).

In my case, I run a nightly de-dupe process on the server that deletes any photos in the auto-upload directory that exist elsewhere in my nextcloud storage, so that any photos which I haven't saved someplace specific are always kept so they can be moved to the appropriate location later rather than lost (or missed).
